scene/gui/label.cpp

```diff
@@ -209,7 +209,7 @@ bool Label::_shape() {
 
 				// Fill after min_size calculation.
 
-				int visible_lines = lines_rid.size();
+				int visible_lines = get_visible_line_count();
 				if (max_lines_visible >= 0 && visible_lines > max_lines_visible) {
```

Comment:
Makes sense, but then also remove the `if` block below, given `get_visible_line_count()` already applies `max_lines_visible`.

Reply:
Oops, missed that. Fixed.